### DISCUSSION TITLE
School isn't listed on the school campuses list

### DISCUSSION THREAD

**Discussion by imamangoo** [community/community#38408] [GitHub Education]
When requesting for a student pack, GitHub requires selecting a school from the "list of school campuses near to you". However, that map doesn't include the school I am current studying in. Any solution?

**Comment by david-engelmann** [comment]
[You can reach out by filling out this form!](https://support.github.com/contact/education)

**Comment by samus-aran** [comment]
Hi @imamangoo 👋 

We recommend that you reach out to your school and ask them to apply and verify the institution. If successful it should update the selection drop down including your verified school to proceed with.
